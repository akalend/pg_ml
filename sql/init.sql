CREATE TABLE astra (
    alpha double precision,
    delta double precision,
    u double precision,
    g double precision,
    r double precision,
    i double precision,
    z double precision,
    run_id bigint,
    cam_col bigint,
    field_id bigint,
    spec_obj_id double precision,
    redshift double precision,
    plate bigint,
    mjd bigint,
    fiber_id bigint
);

COPY astra FROM '/tmp/model/astra.csv' HEADER CSV; 

SELECT * FROM astra LIMIT 2;

CREATE TABLE public.titanic (
    id integer,
    passenger_id integer,
    pclass integer,
    name text,
    sex text,
    age double precision,
    sibsp integer,
    parch integer,
    ticket text,
    fare double precision,
    cabin text,
    embarked character(1),
    res boolean
);

COPY titanic FROM '/tmp/model/titanic.csv' HEADER CSV; 
SELECT * FROM titanic LIMIT 2;
