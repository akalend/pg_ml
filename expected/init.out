CREATE TABLE astra (
    alpha double precision,
    delta double precision,
    u double precision,
    g double precision,
    r double precision,
    i double precision,
    z double precision,
    run_id bigint,
    cam_col bigint,
    field_id bigint,
    spec_obj_id double precision,
    redshift double precision,
    plate bigint,
    mjd bigint,
    fiber_id bigint
);
COPY astra FROM '/tmp/model/astra.csv' HEADER CSV; 
SELECT * FROM astra LIMIT 2;
      alpha       |      delta       |    u     |    g     |    r     |    i     |    z     | run_id | cam_col | field_id |      spec_obj_id       |   redshift   | plate |  mjd  | fiber_id 
------------------+------------------+----------+----------+----------+----------+----------+--------+---------+----------+------------------------+--------------+-------+-------+----------
 16.9568897845004 | 3.64613008870454 | 23.33542 | 21.95143 | 20.48149 |   19.603 | 19.13094 |   7712 |       6 |      442 |  4.855016555329904e+18 |    0.5062369 |  4312 | 55511 |      495
 240.063240247767 | 6.13413059813973 | 17.86033 | 16.79228 | 16.43001 | 16.30923 | 16.25873 |   3894 |       1 |      243 | 2.4489280322708705e+18 | 0.0003448142 |  2175 | 54612 |      348
(2 rows)

CREATE TABLE public.titanic (
    id integer,
    passenger_id integer,
    pclass integer,
    name text,
    sex text,
    age double precision,
    sibsp integer,
    parch integer,
    ticket text,
    fare double precision,
    cabin text,
    embarked character(1),
    res boolean
);
COPY titanic FROM '/tmp/model/titanic.csv' HEADER CSV; 
SELECT * FROM titanic LIMIT 2;
 id | passenger_id | pclass |               name               |  sex   | age  | sibsp | parch | ticket |  fare  | cabin | embarked | res 
----+--------------+--------+----------------------------------+--------+------+-------+-------+--------+--------+-------+----------+-----
  0 |          892 |      3 | Kelly, Mr. James                 | male   | 34.5 |     0 |     0 | 330911 | 7.8292 | -999  | Q        | f
  1 |          893 |      3 | Wilkes, Mrs. James (Ellen Needs) | female |   47 |     1 |     0 | 363272 |      7 | -999  | S        | f
(2 rows)

